client/src/components/teamPage/TeamPage.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from "react";
+import { Box } from "@mui/material";
+import "../Styles/MainPage.scss";
+import { SearchBar } from "../SearchBar";
+import Modal from "react-modal";
+import { Typography } from "@mui/material";
+
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemAvatar from "@mui/material/ListItemAvatar";
+import ListItemText from "@mui/material/ListItemText";
+import Divider from "@mui/material/Divider";
+import Button from "@mui/material/Button";
+import CloseIcon from "@mui/icons-material/Close";
+import PetyaPhoto from "src/images/Petya.jpg";
+import DimitrinaPhoto from "src/images/Dimitrina.jpg";
+import SimeonPhoto from "src/images/Simeon.jpg";
+import IvayloPhoto from "src/images/Ivaylo.jpg";
+import ZahariPhoto from "src/images/Zahari.jpg";
+import GeorgiPhoto from "src/images/Georgi.jpg";
+import PavelPhoto from "src/images/Pavel.jpg";
+import BozhidarPhoto from "src/images/Bozhidar.jpg";
+
+const team = [
+  {
+    name: "Petya Marinova",
+    avatar: PetyaPhoto,
+    personInfo: ["Petya Marinova"],
+    role: "Front - End",
+  },
+  {
+    name: "Dimitrina Yordanova",
+    avatar: DimitrinaPhoto,
+    personInfo: ["Dimitrina Yordanova"],
+    role: "Front - End",
+  },
+  {
+    name: "Simeon Cholakov",
+    avatar: SimeonPhoto,
+    personInfo: ["Simeon Cholakov"],
+    role: "Front - End",
+  },
+  {
+    name: "Ivaylo Slavchev",
+    avatar: IvayloPhoto,
+    personInfo: ["Ivaylo Slavchev"],
+    role: "Front - End",
+  },
+  {
+    name: "Zahari Cheyrekov",
+    avatar: ZahariPhoto,
+    personInfo: ["Zahari Cheyrekov"],
+    role: "Front - End",
+  },
+  {
+    name: "Georgi Peev",
+    avatar: GeorgiPhoto,
+    personInfo: ["Georgi Peev"],
+    role: "Back - End",
+  },
+  {
+    name: "Pavel Pindarev",
+    avatar: PavelPhoto,
+    personInfo: ["Pavel Pindarev"],
+    role: "Back - End",
+  },
+  {
+    name: "Bozhidar Valkov",
+    avatar: BozhidarPhoto,
+    personInfo: ["Bozhidar Valkov"],
+    role: "Back - End",
+  },
+];
+
+export const TeamPage = () => {
+  const [modalIsOpen, setModalIsOpen] = useState(false);
+  const [modalData, setModalData]: any = useState(null);
```

Comment:
Avoid using any as a type, here you will need either null or object I think

Reply:
Fixed.